src/main/java/seedu/address/commons/core/item/Item.java

```diff
@@ -210,16 +210,26 @@ public int hashCode() {
     public String toString() {
         final StringBuilder builder = new StringBuilder();
         builder.append("Description: ")
-                .append(itemDescription.toString())
-                .append(" Task: ")
-                .append(getTask().toString())
-                .append(" Event: ")
-                .append(getEvent().toString())
-                .append(" Reminder: ")
-                .append(getReminder().toString())
-                .append(" Tags: ")
-                .append(getTags().toString());
-        //getTags().forEach(builder::append);
+                .append(itemDescription.toString());
+
+        if (getTask().isPresent()) {
+            builder.append(" Task: ")
+                    .append(getTask().toString());
```

Comment:
I think we should have all the task, event and reminder on a new line so that it will be easier to see. At the same time, we might not need to see every detail within the task, or we might choose to make the done status of a task in a tick and cross fashion (similar to Duke), so we might want to change the toString method for task, event and reminder as well as it is very cluttered at the moment.

Reply:
Actually just put the "\n" for each append :P then it won't look ugly in the result display already

Reply:
Amended!